https://ci.ros2.org/job/ci_windows/23193/console windows is not happy with `C:\ci\ws\src\ros2\rcl\rcl\include\rcl/logging_rosout.h(41,95): error C2099: initializer is not a constant (compiling source file C:\ci\ws\src\ros2\rcl\rcl\src\rcl\logging.c)`.

--- a/rcl/include/rcl/logging_rosout.h
+++ b/rcl/include/rcl/logging_rosout.h
@@ -38,7 +38,18 @@ extern "C"
  * - lifespan = {10, 0}
  */
 RCUTILS_DEPRECATED_WITH_MSG("use rmw_qos_profile_rosout_default instead")
-static const rmw_qos_profile_t rcl_qos_profile_rosout_default = rmw_qos_profile_rosout_default;
+static const rmw_qos_profile_t rcl_qos_profile_rosout_default =
+{
+  RMW_QOS_POLICY_HISTORY_KEEP_LAST,
+  1000,
+  RMW_QOS_POLICY_RELIABILITY_RELIABLE,
+  RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL,
+  RMW_QOS_DEADLINE_DEFAULT,
+  {10, 0},
+  RMW_QOS_POLICY_LIVELINESS_SYSTEM_DEFAULT,
+  RMW_QOS_LIVELINESS_LEASE_DURATION_DEFAULT,
+  false
+};
 
 /// Initializes the rcl_logging_rosout features
 /**

